LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2017
```

Comment:
I guess this is the really initial, before any customization, since it doesn't even list us as the copyright holders yet.